cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -20,6 +20,7 @@ function prepare_config() {
 master_ip=$(_main_ip)
 kubeconfig=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubeconfig
 kubectl=${BASE_PATH}/$KUBEVIRT_PROVIDER/.kubectl
+gocli=${BASE_PATH}/../cluster-up/cli.sh
```

Comment:
hm why do you need it?

Reply:
to copy audit log file from cluster, it is required by REST api coverage (at least for now, this might change in the future) https://github.com/kubevirt/kubevirt/pull/2208/files#diff-a5a178b4c66835110ceff63c61d28c96R483